client/src/components/trip/tripCardStyle.module.css

```diff
@@ -1,9 +1,7 @@
 .tripItem {
-  display: flex;
+  /* display: flex;
   flex-direction: row;
-  justify-content: center;
-}
-
-.avatar div div {
-  margin-top: 5px;
+  justify-content: center; */
+  /* border: 1px solid rgba(0, 0, 0, 0.12); */
+  /* margin-bottom: 20px; */
```

Comment:
you can delete all unused comments

Reply:
I cleaned the code, also wipe out a few console.log so i will add this in the next PR